src/System.Private.Xml/tests/Xslt/XslCompiledTransformApi/XsltArgumentList.cs

```diff
@@ -3019,27 +3019,15 @@ public void AddExtObject24()
         [InlineData(XslInputType.Navigator, ReaderType.XmlValidatingReader, OutputType.Writer, NavType.XPathDocument)]
         [InlineData(XslInputType.Navigator, ReaderType.XmlValidatingReader, OutputType.TextWriter, NavType.XPathDocument)]
         [Theory]
-        public void AddExtObject25(XslInputType xslInputType, ReaderType readerType, OutputType outputType, NavType navType)
+        public void TC_ExtensionObj_Function_Mismatch_IncorrectCasing(XslInputType xslInputType, ReaderType readerType, OutputType outputType, NavType navType)
         {
             MyObject obj = new MyObject(25, _output);
             m_xsltArg = new XsltArgumentList();
 
             m_xsltArg.AddExtensionObject(szDefaultNS, obj);
-
-            if (LoadXSL("MyObject_CaseSensitive.xsl", xslInputType, readerType) == 1)
-            {
-                try
-                {
-                    Transform_ArgList("fruits.xml", outputType, navType);
-                    CheckResult(419.3031944636, outputType);
-                }
-                catch (System.Xml.Xsl.XsltException)
-                {
-                    return;
-                }
-            }
-            _output.WriteLine("Exception not thrown for wrong-case spelling of methods");
-            Assert.True(false);
+            Assert.True(LoadXSL("MyObject_CaseSensitive.xsl", xslInputType, readerType) == 1, "LoadXSL failed");
```

Comment:
Got it. This and other methods on the tests always return 1 or throw an exception so I was actually tempted to transform them to `void` but then the change was going to increase a lot and I am focusing in dealing with the tests that were being skipped.